server/fleet/app.go

```diff
@@ -52,6 +53,9 @@ type AppConfigService interface {
 
 	// License returns the licensing information.
 	License(ctx context.Context) (*LicenseInfo, error)
+
+    // FleetConfig return the config.FleetConfig instance.
+	FleetConfig(ctx context.Context) config.FleetConfig
```

Comment:
I wonder if it makes sense to have this return `(config.FleetConfig, error)` in case there's ever a scenario where some config needs to be loaded?